### PR TITLE
Adds Industry Agglomeration Demo

### PR DESCRIPTION
**Pull request by AlexandreSajus**
Adds a new demo about displaying industry agglomerations in the US using a map and a chart. This demo showcases Taipy 3.1's ability to interact with all Plotly charts

image

https://industry-agglom.taipy.cloud/
https://github.com/Avaiga/demo-industry-agglom

Slight issue:
The thumbnail of the demo in the gallery pages does not show when I build the docs despite the docs pointing to the right path. Is this just a local issue? How do I fix this?
image

### CODE REVIEW

**Comment by FlorianJacta** [comment]
I don't see your issue. Try to rebuild it maybe.

File: mkdocs.yml_template [hunk 1]
```diff
@@ -215,6 +215,7 @@ nav:
     - "Visualization":
       - "Visualization": gallery/visualization/index.md
       - "COVID Dashboard": gallery/visualization/2_covid_dashboard/index.md
+      - "Measuring Industry Agglomeration": gallery/visualization/industry_agglom/index.md
```
Comment (FlorianJacta) [inline mkdocs.yml_template]: The order of cards on the Visualization page differs from the navigation to the left. You can change the navigation or the order of the files by changing the folder name. The cards are ordered alphabetically by their corresponding folders

**Review by FlorianJacta — CHANGES_REQUESTED** [review]

File: mkdocs.yml_template (hunk 1 above)
Comment (FlorianJacta) [reply mkdocs.yml_template]: image

Comment (AlexandreSajus) [reply mkdocs.yml_template]: Fixed by adding "7_" to the folder name

**Review by FlorianJacta — APPROVED** [review]
👍

**Pull request merged**